### DISCUSSION TITLE
Consider using FastEndpoints instead of Minimal API

### DISCUSSION THREAD

**Discussion by afroukh** [jasontaylordev/CleanArchitecture#1168] [Ideas]
it renders developing api more faster by generating request and response

**Comment by victorykj** [comment]
Has anyone figured out how to integrate fastendpoints with these templates

**Comment by jasontaylordev** [comment]
Thanks for the suggestion! Sticking with ASP.NET Core Minimal API for now to keep the template simple and approachable. It offers enough flexibility for most use cases without adding extra complexity.
